lib/extensions/generate_palette.py

```diff
@@ -84,4 +84,4 @@ def _get_color_from_elements(self, elements):
 
 if __name__ == '__main__':
     e = GeneratePalette()
-    e.affect()
+    e.effect()
```

Comment:
Are you 100% sure about this?  At least in the old version of inkex, there was a big difference between `affect()` and `effect()`...

Reply:
Looks like we want `.run()`, I think?
 https://inkscape-extensions-guide.readthedocs.io/en/latest/_modules/inkex/deprecated.html#DeprecatedEffect.affect